pkg/entry/mongo/document.go

```diff
@@ -121,13 +122,19 @@ func (d *JobLogDocument) Populate(ctx context.Context, ts time.Time, record map[
 	return d.generateObjectID()
 }
 
+func cleanLogContent(content string) string {
+	endOfLineRegexp := regexp.MustCompile(`(\n)+$`)
+	return endOfLineRegexp.ReplaceAllString(content, "")
+}
+
 func (d *LogDocument) Populate(ctx context.Context, ts time.Time, record map[interface{}]interface{}) error {
 	logger, err := log.GetLogger(ctx)
 	if err != nil {
 		return fmt.Errorf("get logger: %w", err)
 	}
 
-	d.Log, err = parse.ExtractStringValue(record, LogKey)
+	logContent, err := parse.ExtractStringValue(record, LogKey)
+	d.Log = cleanLogContent(logContent)
 	if err != nil {
 		if !errors.Is(err, &parse.ErrKeyNotFound{
 			LookingFor: LogKey,
```

Comment:
Never ever use a value before checking the associated error!

Reply:
No excuse to put this line here, I agree :D